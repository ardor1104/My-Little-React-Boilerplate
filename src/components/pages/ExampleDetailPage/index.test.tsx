import { composeStories } from '@storybook/testing-react';
import { act } from '@testing-library/react';

import useRender from 'hooks/useRender';

import * as stories from './index.stories';

const { Default } = composeStories(stories);

describe('pages/ExampleDetailPage', () => {
  it('render Default', async () => {
    const component = await act(() => useRender(<Default />));

    expect(component).toMatchSnapshot();
  });
});
