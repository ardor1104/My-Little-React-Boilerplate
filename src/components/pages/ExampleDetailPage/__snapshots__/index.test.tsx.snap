// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/ExampleDetailPage render Default 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="sc-bcXHqe cgfMzA"
      >
        <header
          class="sc-eDvSVe sc-jSUZER fXyNOr hCldj"
        >
          <div
            class="sc-gKPRtg xFlkb"
          >
            <a
              class="sc-gswNZR dXalQK"
              href="/"
              kind="text"
              role="button"
            >
              MainPage
            </a>
            <a
              class="sc-gswNZR dXalQK"
              href="/example"
              kind="text"
              role="button"
            >
              ExamplePage
            </a>
          </div>
        </header>
        <div
          class="sc-eDvSVe fXyNOr"
        />
        <div
          class="sc-pyfCe"
        >
          ExampleDetailPage
          [undefined]
        </div>
        <div
          class="sc-iBYQkv aRxZQ"
        >
          <p
            class="sc-ftTHYK EzcYq"
          >
            1
          </p>
          <button
            class="sc-hLBbgP bFSXPk"
            kind="mono"
            role="button"
          >
            - Decrease
          </button>
          <button
            class="sc-hLBbgP bFSXPk"
            kind="mono"
            role="button"
          >
            + Increase
          </button>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="sc-bcXHqe cgfMzA"
    >
      <header
        class="sc-eDvSVe sc-jSUZER fXyNOr hCldj"
      >
        <div
          class="sc-gKPRtg xFlkb"
        >
          <a
            class="sc-gswNZR dXalQK"
            href="/"
            kind="text"
            role="button"
          >
            MainPage
          </a>
          <a
            class="sc-gswNZR dXalQK"
            href="/example"
            kind="text"
            role="button"
          >
            ExamplePage
          </a>
        </div>
      </header>
      <div
        class="sc-eDvSVe fXyNOr"
      />
      <div
        class="sc-pyfCe"
      >
        ExampleDetailPage
        [undefined]
      </div>
      <div
        class="sc-iBYQkv aRxZQ"
      >
        <p
          class="sc-ftTHYK EzcYq"
        >
          1
        </p>
        <button
          class="sc-hLBbgP bFSXPk"
          kind="mono"
          role="button"
        >
          - Decrease
        </button>
        <button
          class="sc-hLBbgP bFSXPk"
          kind="mono"
          role="button"
        >
          + Increase
        </button>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
